app/views/metadata_fields/_statutory_instruments.html.erb

```diff
@@ -1,55 +1,15 @@
-<%= render "shared/legacy/date_fields_legacy", f: f, field: :sift_end_date, format: :statutory_instrument %>
-<%= render "shared/legacy/date_fields_legacy", f: f, field: :laid_date, format: :statutory_instrument %>
-<%= render "shared/form_group", f: f, field: :sifting_status do %>
-  <%= f.select :sifting_status, select_options_for_facet(f.object.allowed_values(:sifting_status)), {}, { class: "form-control" } %>
+<%= render layout: "shared/specialist_document_form", locals: { f: f } do %>
 <% end %>
-<% # TODO: this bit of logic prevents us moving to the shared view %>
+
+<% format_name = f.object.class.document_type.to_sym %>
 <% withdrawn_date_display = f.object.withdrawn_sifting_status? ? "block" : "none" %>
```

Comment:
I disagree with the ordering... I think Withdrawn Date needs to sit directly below Sifting Status as it was like in the old form. The reason being I believe it looks like it's connected, as the Sifting status affects whether or not Withdrawn date even appears: 

<img width="810" alt="image" src="https://github.com/user-attachments/assets/925c05f2-4061-4c34-92e4-6ae98596f41c" />